packages/common/src/client/clamav-rest-client.js

```diff
@@ -0,0 +1,53 @@
+const axios = require('axios').default;
+const FormData = require('form-data');
+// const { Readable } = require('stream');
+const fs = require('fs');
+
+const trailingSlashRegex = /\/$/;
+
+class ClamAVClient {
+	/**
+	 * @param {string} host - e.g. https://example.com
+	 * @param {number} [maxBodyLength] - maxBodySize in bytes used by axios
+	 */
+	constructor(host, maxBodyLength) {
+		if (!host) {
+			throw new Error('host is required');
+		}
+
+		/** @type {string} */
+		this.host = host.replace(trailingSlashRegex, '');
+		this.maxBodyLength = maxBodyLength ? maxBodyLength * 1.1 : maxBodyLength; // add headroom
```

Comment:
something was larger in the forwarded request than the incoming one so added some headroom, feels hacky if someone can spot what would likely be increasing the request size could make more specific